discord-scripts/invite-management.ts

```diff
@@ -226,7 +226,7 @@ export default async function sendInvite(discordClient: Client, robot: Robot) {
             await internalChannel.send(
               `@here **Welcome to the ${clientName.value} Internal Audit Channel!**\n` +
                 `You can use this invite to access <#${internalChannel.id}>: \`${internalInvite.url}\`\n` +
-                `This invite will expire **on <t:${internalInviteExpiry}:R>** and has **${internalInvite.maxUses} max uses**`,
+                `This invite will expire <t:${internalInviteExpiry}:R> and has **${internalInvite.maxUses} max uses**`,
```

Comment:
Awesome! A note on the “best way” here: discord.js has a [`time`](https://discord.js.org/docs/packages/formatters/main/time:Function) function that can take a number and a [`TimestampStyle`](https://discord.js.org/docs/packages/formatters/main/TimestampStyles:Variable) to be a little clearer on what we're doing (it's overload 4).